app/controllers/front_ui_controller.rb

```diff
@@ -44,6 +44,20 @@ def save_location
 
   def about; end
 
+  def location
+    @loc = params[:loc].downcase
+
+    @maps = Map.page(params[:maps])
+               .per_page(20)
+               .where(archived: false, password: '', location: "%#{@loc}%")
```

Comment:
Its not working

Reply:
I fixed it. I am really sorry about all this. I will be careful from next time.

Reply:
Don't be sorry. It's fine, take your time.

Reply:
I am getting 👇 locally.

<img width="1617" alt="Screenshot 2019-08-18 at 12 05 54 AM" src="https://user-images.githubusercontent.com/20878070/63215992-14135f80-c14c-11e9-8a25-ba043dd5ea20.png">